scapy/main.py

```diff
@@ -65,26 +68,33 @@ def _usage():
 ######################
 
 
-def _load(module):
+def _load(module, globals_dict=None, symb_list=None):
```

Comment:
`"""Loads a Python module to make variables, objects and functions available globally. The logic is to load the module using importlib, then copy the symbols to the global symbol table"""`